install/kubernetes/istio-initializer.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-inject
+  namespace: istio-system
+data:
+  config: |-
+    policy: "enabled"
+    namespaces: [""] # everything, aka v1.NamepsaceAll, aka cluster-wide
+    initializerName: "sidecar.initializer.istio.io"
```

Comment:
Do we keep sidecar here?

Reply:
That's up to. It requires another corresponding change in istio/pilot